tests/cursor/jsfmt.spec.js

```diff
@@ -30,7 +36,9 @@ test("doesn't insert second placeholder for nonexistent TypeAnnotation", () => {
 foo('bar', cb => {
   console.log('stuff')
 })`;
-  expect(prettier.formatWithCursor(code, { cursorOffset: 24 })).toEqual({
+  expect(
+    prettier.formatWithCursor(code, { parser: "babylon", cursorOffset: 24 })
```

Comment:
It's worth noting that you now *must* specify a parser when using the API, or your code will be passed through as-is. This might be counter-intuitive, and is a bit more of a breaking change than the CLI fix. We might want to default to babylon, but *only* when using the API directly.

Reply:
I think we can go back to #2884 and ask/ping if anyone's using the API without both `filepath` and `parser` and relying on the babylon fallback